encord/orm/dataset.py

```diff
@@ -120,14 +211,180 @@ def created_at(self, value: datetime) -> None:
         """Datetime will trim milliseconds for backwards compatibility."""
         self["created_at"] = value.strftime(DATETIME_STRING_FORMAT)
 
+    @property
+    def querier(self) -> Querier:
+        return self["querier"]
+
+    @querier.setter
+    def querier(self, new_querier: Querier) -> None:
+        self["querier"] = new_querier
+
+    @property
+    def frames_per_second(self) -> int:
+        if self.data_type != DataType.VIDEO:
+            return None
+        return self["frames_per_second"]
+
+    @frames_per_second.setter
+    def frames_per_second(self, new_frames_per_second: int) -> None:
+        self["frames_per_second"] = new_frames_per_second
+
+    @property
+    def duration(self) -> Optional[int]:
+        if self.data_type != DataType.VIDEO:
+            return None
+        return self["duration"]
+
+    @duration.setter
+    def duration(self, new_duration: int) -> None:
+        self["duration"] = new_duration
+
     @property
     def client_metadata(self) -> Optional[dict]:
         """
         Custom client metadata. This is null if it is disabled via the
         :class:`encord.orm.dataset.DatasetAccessSettings`
         """
+        if self["client_metadata"] is None:
```

Comment:
This breaks the current client usage. Can you please add a `DEPRECATED` warning here (you can search for similar examples in our code). 

Then, just have `get_client_metadata` and `set_client_metadata` function instead of this outdated one.